.github/workflows/release.yml

```diff
@@ -72,4 +72,21 @@ jobs:
           token: ${{ secrets.APP_SERVICES_CI }}
           repository: redhat-developer/app-services-guides
           event-type: rhoas-cli-release
-          client-payload: '{"version": steps.set-version.outputs.version }'
\ No newline at end of file
+          client-payload: '{"version": steps.set-version.outputs.version }'
+  quay-release:
+    if: steps.check-tag.outputs.prerelease == 'false'
+    needs: release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Notify tools repo
+        uses: actions/github-script@v6
+        with: 
+          script: |
+            github.request('POST /repos/redhat-developer/app-services-tools/actions/workflows/20880234/dispatches', {
+              workflow_id: 20880234,
```

Comment:
Workflow ID from here: https://api.github.com/repos/redhat-developer/app-services-tools/actions/workflows/20880234

Reply:
Better way of triggering jobs?

Reply:
it is different to repository dispatch, not sure if it is objectively better or not. It is the reason we can trigger manually with a form and also can be triggered on non-main branches so it is a bit more flexible.

Reply:
Reason I'm asking is that I would need to setup another repo for that today and was wondering which job to use. 
Will try this now. Thank you